cre.cpp

```diff
@@ -3627,6 +3659,10 @@ static const struct luaL_Reg credocument_meth[] = {
     {"saveDefaults", saveDefaults},
     {"close", closeDocument},
     {"__gc", closeDocument},
+    {"setUserHyphenationDict", setUserHyphenationDict},
+    {"getHyphenationForWord", getHyphenationForWord},
+    {"getLowercasedWord", getLowercasedWord},
+    {"formatHyphenationSuggestion", formatHyphenationSuggestion},
```

Comment:
Please move them just after hasNonLinearFlows (the last real added feature), from readDefaults to __gc it's mostly low-level internal stuff.